src/app/components/activity/feed/post/AppActivityFeedPostHeader.vue

```diff
@@ -0,0 +1,135 @@
+<script lang="ts" setup>
+import { computed, PropType } from 'vue';
+import { FiresidePost } from '../../../../../_common/fireside/post/post-model';
+import { Screen } from '../../../../../_common/screen/screen-service';
+import { AppTimeAgo } from '../../../../../_common/time/ago/ago';
+import AppUserCardHover from '../../../../../_common/user/card/hover/hover.vue';
+import AppUserFollowWidget from '../../../../../_common/user/follow/widget.vue';
+import AppUserAvatar from '../../../../../_common/user/user-avatar/user-avatar.vue';
+import AppUserVerifiedTick from '../../../../../_common/user/verified-tick/verified-tick.vue';
+import { ActivityFeedView } from '../view';
+import AppActivityFeedPostTime from './time/time.vue';
+
+const props = defineProps({
+	post: {
+		required: true,
+		type: Object as PropType<FiresidePost>,
+	},
+	feed: {
+		default: null,
+		type: (null || Object) as PropType<ActivityFeedView | null>,
```

Comment:
`null || Object` doesn't make sense here for type, right? Wouldn't vue just use Object?

Reply:
I think you do type: [null, Object]? Or you just say it's not required with a default of null and it should work, no?